package.json

```
@@ -21,6 +21,7 @@
     "eslint-fix": "eslint --fix . --report-unused-disable-directives",
     "generate": "run-s generate:*",
     "generate:imports": "node tools/generate-imports.mjs",
+    "ubuntu-distro-info": "node tools/distro-json-generate.mjs",
```

```suggestion
    "update:ubuntu-distro-info": "node tools/distro-json-generate.mjs",
```

Let' suse `update:` prefix, please reorder alphanumeric.

---

done